web/js/roomController.js

```diff
@@ -795,11 +795,17 @@ RecordingsController, ScreenShareController, FeedbackController, PhoneNumberCont
               }
             };
           };
-        loadAnnotations = LazyLoader.load([
-          '/js/vendor/jquery-3.3.1.min.js',
-          'https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js',
-          '/js/vendor/opentok-annotation.js'
-        ]);
+        var annotationsLibs = [];
+
+        if(aParams.useGoogleApi)
+          annotationsLibs.push('//ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js');
+        else
+          annotationsLibs.push('//static.opentok.com/js/vendor/jquery-3.3.1.min.js');
```

Comment:
Again, it would be nice if we could remove jQuery entirely. But the opentok-annotation.js file uses jQuery a few times, and I haven't looked into how to replace the `$.Deferred()` call it uses.

Reply:
Thanks for the feedback @jeffswartz, I think for the moment to remove JQuery is not an option. Regarding the `JQUERY_URL` another solution could be to use the same CDN that we are using to load `underscore-min.js`, why don't we do that instead of add another config setting?

Reply:
If cdnjs.cloudflare.com loads in China, should we simply replace the ajax.googleapis.com URL with the cloudflare URL (and remove the config setting entirely)?

Also, this config setting would not work with `//static.opentok.com/js/vendor/jquery-3.3.1.min.js` (if we ever wanted to use that).

Reply:
> If cdnjs.cloudflare.com loads in China, should we simply replace the ajax.googleapis.com URL with the cloudflare URL (and remove the config setting entirely)?
Yes, correct, in my opinion it's a simpler solution

> Also, this config setting would not work with //static.opentok.com/js/vendor/jquery-3.3.1.min.js (if we ever wanted to use that).
Because of the route structure? I changed the file route in our CDN to have the same structure of the jquery files served by googleapis and cloudflare: https://static.opentok.com/js/vendor/jquery/3.3.1/jquery.min.js